ftl/node/main.py

```diff
@@ -90,4 +91,4 @@ def main():
 
 
 if __name__ == '__main__':
-    main()
+    main(sys.argv)
```

Comment:
One more tweak, this is a bit easier to use if you pass argv[1:] here, then you don't need the "main.py" in your test.

Reply:
This all makes a lot more sense, thanks Dan!